shared/js/background/email-utils.js

```diff
@@ -165,6 +166,15 @@ export const sendJSPixel = (options) => {
     case 'incontext_close_x':
         fireIncontextSignupPixel('incontext_close_x_extension')
         break
+    case 'incontext_eligible': {
+        const shouldFireIncontextEligibilityPixel = getSetting('shouldFireIncontextEligibilityPixel')
+        if (!shouldFireIncontextEligibilityPixel) return
+
+        const daysSinceInstallation = Math.ceil(daysInstalled())
```

Comment:
We do `Math.ceil` because this will be used for a cutoff, so we're interested in the upper bound, not the usual rounding.